connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -103,6 +117,11 @@ public static SdkProfile getSdkProfile() {
         return sdkProfile;
     }
 
+    public static void beforeAuthentication() {
+        logSessionId = UUID.randomUUID().toString();
+        tsLoginButtonClicked = System.currentTimeMillis();
+    }
+
     public static synchronized void authenticate(
```

Comment:
For this `authenticate` method, should we do `parameters.put("lsi", getLogSessionId())`, so it gets passed along?

Reply:
Done. Put in the shared authenticate method